EIPS/eip-2200.md

```diff
@@ -0,0 +1,96 @@
+---
+eip: 2200
+title: Rebalance net-metered SSTORE gas cost with consideration of SLOAD gas cost change
+author: Wei Tang (@sorpaas)
+discussions-to: https://github.com/sorpaas/EIPs/issues/1
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-07-18
+requires: 1283, 1706, 1884
```

Comment:
So is this requiring 1283/1706 or replacing them?

Reply:
I don't see how it requires `1283` ? 
And I certainly don't see how it requires `1706`... ?

Reply:
It requires all of the three EIPs to be functional.

Reply:
Fixed.